src/rpc/core_rpc_server.cpp

```diff
@@ -3008,26 +3008,44 @@ namespace cryptonote { namespace rpc {
 
     std::vector<std::string> args;
 
-    uint64_t const curr_height   = m_core.get_current_blockchain_height();
-    uint64_t staking_requirement = service_nodes::get_staking_requirement(nettype(), curr_height);
+    std::optional<uint64_t> height = m_core.get_current_blockchain_height();
+    auto hf_version = get_network_version(nettype(), *height);
+    uint64_t staking_requirement = service_nodes::get_staking_requirement(nettype(), *height);
 
     {
       uint64_t portions_cut;
-      if (!service_nodes::get_portions_from_percent_str(req.operator_cut, portions_cut))
+      if (hf_version >= hf::hf19_reward_batching)
```

Comment:
I don't think we need this `if` anymore; we can *always* assume ≥hf19 rules now.

(Actually, come to think of it, there's probably some delete-me-after-hf19 code we can remove as well).

Reply:
I've just pushed code to remove pre-hf19 regs from `oxend prepare_registration`